spring-ai-alibaba-examples/function-calling-example/src/main/resources/application.yml

```diff
@@ -1,7 +1,14 @@
 spring:
   application:
-    name: cfunction-calling-example
+    name: function-calling-example
 
   ai:
     dashscope:
       api-key: ${AI_DASHSCOPE_API_KEY}
+
+
+plugins:
+  dingTalk:
+    customRobot:
+      accessToken: ${plugins.dingTalk.customRobot.customRobot.accessToken} # accessToken of custom robot
+      signature: ${plugins.dingTalk.customRobot.customRobot.signature}     # sign of custom robot
```

Comment:
I think it's better to use the following configuration structure. To achieve that, we can define a `DingtalkPluginProperties` to put all configuration properties. Like suggested here https://github.com/alibaba/spring-ai-alibaba/pull/112#discussion_r1849413353

```yaml
spring:
  ai:
    plugins:
       dingTalk: 
          customRobot:
            accessToken:
            signature:
```

Reply:
yes sir